VMBackup/main/Utils/EventLoggerUtil.py

```diff
@@ -173,18 +168,19 @@ def _process_events(self):
                     return
             if not self.event_queue.empty():
                 if sys.version_info[0] == 2:
-                    event_file_path = os.path.join(self.temporary_directory, "{}.json".format(int(time.time() * 1000000000)))
+                    event_file_path = os.path.join(self.events_folder, "{}.json".format(int(time.time() * 1000000000)))
```

Comment:
can you please make this change behind a flag to switch to older implementation since new implementation requires testing?